Utils/github_workflow_scripts/handle_external_pr.py

```diff
@@ -253,13 +255,56 @@ def is_requires_security_reviewer(pr_files: list[str]) -> bool:
     return False
 
 
-def is_tim_content(pr_files: list[str]) -> bool:
+def check_new_pack_metadata(pr_files: list[str], external_pr_branch: str, repo_name: str) -> bool:
+    """
+    Checks if tim reviewer needed, if the pack is new and not part of Master
+
+    Arguments:
+        - `pr_files`: ``List[str]``: The list of files changed in the Pull Request. Will be used to determine
+        whether a security engineer is required for the review.
+        - 'external_pr_branch': str : name of branch to checkout
+        - 'repo_name': str : name of repository
+
+    Returns: `bool` whether a security engineer should be assigned
+    """
+    pack_dirs_to_check = packs_to_check_in_pr(pr_files)
+    try:
+        fork_owner = os.getenv('GITHUB_ACTOR')
+        with Checkout(
+            repo=Repo(Path().cwd(), search_parent_directories=True),
+            branch_to_checkout=external_pr_branch,
+            # in marketplace contributions the name of the owner should be xsoar-contrib
+            fork_owner=fork_owner if fork_owner != 'xsoar-bot' else 'xsoar-contrib',
+            repo_name=repo_name
+        ):
+            pack_metadata = get_metadata(pack_dirs_to_check)
+            for file in pr_files:
+                if "yml" in file and "Integrations" in file:
+                    content_yml = get_yaml(file_path=file)
+                    is_feed = content_yml.get("script").get("feed", "False")
+                    print(f'Is it a feed: {is_feed}')
+                    if is_feed:
+                        return True
+    except Exception as er:
+        print(f"couldn't checkout branch to get metadata, error is {er}")
+        return False
+    print(f'the metadata is: {pack_metadata}')
+    tags = pack_metadata.get("tags")
+    categories = pack_metadata.get("categories")
+    if TIM_TAGS in tags or TIM_CATEGORIES in categories:  # type: ignore
+        return True
+    return False
+
+
+def is_tim_content(pr_files: list[str], external_pr_branch: str, repo_name: str) -> bool:
     """
     This is where the actual search for feed:True or relevant tags or categories are being searched
     according to the login in is_tim_reviewer_needed
 
     Arguments:
     - pr_files: List[str] The list of files changed in the Pull Request.
+    - 'external_pr_branch': str : name of branch to checkout
+    - 'repo_name': str : name of repository
```

Comment:
Same as in is_tim_reviewer_needed

Reply:
I don't understand what should be the same? the description of the arguments is the same and the relevant arguments are listed.

What should I fix?